lib/tasks/statistics.rake

```diff
@@ -0,0 +1,44 @@
+namespace :statistics do
+  desc "Get information on all form responses for a specified day, e.g. rake statistics:form_responses[\"2020-04-13\"]"
```

Comment:
```suggestion
  desc "Get information on all form responses for a specified day, e.g. rake statistics:form_responses[2020-04-13]"
```

Reply:
Ooh, I hadn't realised you could drop the quotes